syntax = "proto3";

package proto;
option go_package = "bus-project/proto";

message SeatDetection {
    repeated int32 seat_number = 1;
    int64 timestamp = 2;
}

import "google/protobuf/empty.proto";
// message Empty {}

service SeatDetectionService {
    rpc Send(SeatDetection) returns (google.protobuf.Empty) {}
}
